src/expr.rs

```diff
@@ -2149,6 +2262,15 @@ impl From<Expr> for SelectExpr {
     }
 }
 
+impl<T> From<T> for SimpleExpr
+where
+    T: Into<Value>,
```

Comment:
Interesting... I was thinking of the potential drawbacks of this blanket impl but could not think of one.
Should improve ergonomics

Reply:
This is amazing!